e2e/package.json

```diff
@@ -55,5 +55,8 @@
     "test": "start-test start:webhook 3000 test:ci",
     "test:ci": "node run_tests.js",
     "uniq-meta": "grep -r \"^// $META:\" cypress | grep -ow '@\\w*' | sort | uniq"
+  },
+  "dependencies": {
+    "authenticator": "^1.1.5"
```

Comment:
Please use fixed version and move to "devDependencies" as we're putting it all there (no distinction for E2E).